Data editing interface not visible

Steps: open a new or existing data card

Bug: I can't see the div at the top that's supposed to let me edit my data.
![eve-cant-see-input-div_15-10-19](https://cloud.githubusercontent.com/assets/4466164/10829816/a6f39b94-7e52-11e5-9712-08e5570284c8.png)

Gist: https://gist.github.com/anonymous/e051e6bb907d22e9b60a

Environment:
OSX Macbook Pro, 10.9.5
Chrome 46.0.2490.80 (64-bit)
Eve (version file says 0, not sure where else to look, but I cloned it today)

Details:
With a little testing, it seems .form-container's height is somehow set to 2px.

I'm not sure if it's relevant, but when I load any page I get these two errors:
```
microReact.ts:245 Error: Invalid value for <svg> attribute height="-Infinity"
Renderer.domify @  microReact.ts:245
Renderer.render @ microReact.ts:465
UiRenderer.render @ uiRenderer.ts:73
(anonymous function) @ editor.ts:177
```
```
microReact.ts:246 Error: Invalid value for <svg> attribute width="-Infinity"
Renderer.domify @ microReact.ts:246
Renderer.render @ microReact.ts:465
UiRenderer.render @ uiRenderer.ts:73
(anonymous function)	@	editor.ts:177
```

And when I run.sh, I get these warnings:
```
Warning, cannot migrate state for: "562fd764-8fb5-42e5-8cd4-39c4575ea424"
Warning, cannot migrate state for: "562fd764-8fb5-42e5-8cd4-39c4575ea424"
Warning, cannot migrate state for: "562fd764-8fb5-42e5-8cd4-39c4575ea424"
Warning, cannot migrate state for: "562fd764-8fb5-42e5-8cd4-39c4575ea424"
```

--- a/ui/css/editor.css
+++ b/ui/css/editor.css
@@ -163,7 +163,7 @@ body table th * { display: inline-block; }
 - Table workspace
 ---------------------------------------------------------*/
 
-.table-workspace .form-container { width: auto; max-height:70vh; }
+.table-workspace .form-container { width: auto; max-height:70vh; height: 100%; }
 .table-workspace .form { max-height:70vh; }
 .table-workspace .form-fields { flex-wrap:wrap; }
 .table-workspace .field-item { min-width:300px; }

